non-free.md

```diff
@@ -18,8 +18,8 @@
   * [Auth0](https://auth0.com/docs/appliance): `⊘ Proprietary`. Identity made simple for developers. `NodeJS`
 
 ## Content Management Systems (CMS)
-
-  * [Kirby](https://getkirby.com/): `⊘ Proprietary` File-based CMS. Easy to setup. Easy to use. Flexible as hell. `PHP` ([Source Code](https://github.com/getkirby/kirby))
+  * [CraftCMS](https://craftcms.com/) `⊘ Proprietary` - Craft is a content-first CMS that aims to make life enjoyable for developers and content managers alike. `PHP`
```

Comment:
Could you add in the [demo](https://demo.craftcms.com/) between the description and language tag please? Everything else looks good.

Formatting:

    [Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language`